content/en/tracing/enable/_index.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Enable trace collection in Datadog
+kind: Documentation
+aliases:
+  - /tracing/languages/
+  - /tracing/environments/
+  - /tracing/setup/environment
+  - /tracing/setup/first_class_dimensions
+  - /tracing/getting_further/first_class_dimensions/
+  - /agent/apm/
+further_reading:
+- link: "/agent/docker/apm"
+  tag: "Documentation"
+  text: "Docker APM setup"
+- link: "/integrations/amazon_ecs/#trace-collection"
+  tag: "Documentation"
+  text: "ECS EC2 APM setup"
+- link: "/integrations/ecs_fargate/#trace-collection"
+  tag: "Documentation"
+  text: "ECS Fargate APM setup"
+---
+
+To use APM, start by enabling trace collection in Datadog. You can do this in multiple different ways depending on your system setup: including using the [Datadog Agent locally](#Datadog-Agent), [on containers](#Containers), and [several other ways](#Other-ways-to collect-traces). For the full overview of all of the steps to set up APM, see the [APM overivew][1].
+
+## Datadog Agent
+
+APM is enabled by default in Agent 6. Set `apm_non_local_traffic: true` if you are sending traces from a nonlocal environment (like a container). To get an overview of all the possible settings for APM, take a look at the Agent's [`datadog.example.yaml`][2] configuration file. For more information about the Datadog Agent, see the [dedicated doc page][3] or refer to the [`datadog.yaml` templates][4].
+
+Tracing metrics are sent to the Agent from your application: the application code instrumentation sends to the Agent every second (for example, see [the Python client][5]) and the Agent sends to the Datadog API every 10 seconds.
```

Comment:
This is not true. Traces are sent thanks to the application instrumentation libs to the Agent which then calculates the Tracing metrics. (cf previous comment)

Reply:
I think we could just remove this.